src/Voron/Platform/PalFlags.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+// ReSharper disable InconsistentNaming
+// ReSharper disable UnusedMember.Local
+// ReSharper disable IdentifierTypo
+
+namespace Voron.Platform
+{
+    public static class PalFlags
+    {
+        [Flags]
+        public enum Errno
```

Comment:
Let's remove this.
I would rather have a dedicated function in PAL that I can hand an error code and get a string back that would hold the error string.